### ISSUE TITLE
Include useful information in events

### ISSUE THREAD

**Issue by ep1cman** [asantaga/wiserHomeAssistantPlatform#324]
Hi,

I am currently trying to setup notification for when my heating fires up to track the cause. It would be super useful for the events triggered by this integration to include some data to provide context. It seems this is already in the code but for some reason has been commented out:
https://github.com/asantaga/wiserHomeAssistantPlatform/blob/6549411942e8c5e99b6fa63988b109c15fe7b8c6/custom_components/wiser/events.py#L59

In addition, I think the boost duration could be added to the `is_boosted` event via `CONF_EVENT_DATA`

**Comment by msp1974** [comment]
I was going to add event data with these events but then realised that you can get any attributes you want off the climate device by using (as an example)...

```yaml
{{state_attr(trigger.event.data.entity_id, 'temperature')}}
```

Is there a scenario this would not work?

**Comment by ep1cman** [comment]
I am using nodered instead of home assistants built-in scripting (see below). I have a node that listens to the `wiser_event` event and it emmits a message containing the event data. 

I have worked arround this similarly to your above snippet by extracting the `entity_id` and then querying the entity state separately but this is more convoluted, especially given the code is all there already.

<img width="866" alt="image" src="https://user-images.githubusercontent.com/5303409/200150924-9c0b5222-1949-479d-968a-04e1044d3786.png">

**Comment by msp1974** [comment]
Ok that makes sense when using nodered. I'll add some event data for next release.

**Comment by msp1974** [comment]
Added to v3.2.3beta release if you could check it.

**Comment by msp1974** [comment]
@ep1cman  Were you able to check this?

**Comment by ep1cman** [comment]
Hi sorry for the delay, I can confirm that the events are now spitting out additional data as requested. Thanks!

Here is a screenshot of the event being recieved in NodeRED:
![image](https://user-images.githubusercontent.com/5303409/216969146-0cdb8cb0-76f4-437a-98ce-94174cdda632.png)

**Comment by msp1974** [comment]
Great.  Closing issue.  Now formally released in v3.3.0.
